edx_credentials_themes/templates/edx.org/credentials/programs/professional-certificate/certificate.html

```diff
@@ -1,23 +1,26 @@
 {% extends 'edx.org/credentials/programs/base.html' %}
 {% load i18n %}
+{% load html %}
 
 {% block accomplishment_summary %}
-  {% trans "successfully completed all courses and received passing grades for a Professional Certificate in" %}
+  {# Translators: This phrase is preceded by the learner's name and followed by the course title #}
+  {% trans "successfully completed all courses and received passing grades for a Professional Certificate in" as tmsg %}{{ tmsg | htmlescape }}
```

Comment:
I think this is fine for now.  But it is not a reasonable i18n solution for many other languages.  Can you add a summary of what work would need to be done to include name and title within the translation block

https://openedx.atlassian.net/wiki/spaces/LEARNER/pages/161840694/i18n+Tech+Debt